app.js

```diff
@@ -41,6 +41,14 @@ app.use(bodyParser.urlencoded({ extended: true }))
 var routes = require('./routes/main')
 app.use('/', routes)
 
+app.get('/services',function (req, res, next) {
```

Comment:
Ok noted. Now that u understand how to write get endpoints. Next time add it to the correct route. try not to put it in the app.js file